ch_pipeline/hfb/analysis.py

```diff
@@ -351,6 +351,104 @@ def process(self, minuend, subtrahend):
         return out
 
 
+class HFBOnOffDifference(task.SingleTask):
+    """Computes the on-off differencing
+
+    Used for flattening sub-frequency band shape by differencing on-source and
+    off-source data.
+    """
+
+    offset = config.Property(proptype=int, default=5)
+    nsamples = config.Property(proptype=int, default=5)
+
+    def process(self, stream):
+        """Take difference and place in container.
+
+        Parameters
+        ----------
+        stream : HFBHighResRingMap
+            Container with stacked HFB data and weights.
```

Comment:
I think this line is maybe also from a previous task
```
            Container with stacked HFB data and weights.
```

Reply:
Yes, it doesn't necessarily need to be stacked! `HFBHighResRingMap` is a container for HFB data and weights with a single high resolution frequency axis.